### PR TITLE
✨ ✅ JW Player Outstream/Resize Support

### PR DESCRIPTION
**Pull request by zetagame**
- Implement outstream support

- Implement resize support

- Create outstream player example

- Create additional test for parameter swap

### CODE REVIEW

File: extensions/amp-jwplayer/0.1/amp-jwplayer.js [hunk 1]
```diff
@@ -287,6 +287,11 @@ class AmpJWPlayer extends AMP.BaseElement {
     this.contentBackfill_ = element.getAttribute('data-content-backfill') || '';
     this.contentRecency_ = element.getAttribute('data-content-recency') || '';
 
+    // Allow outstream playback
+    if (this.contentid_ === 'outstream') {
```
Comment (kristoferbaxter) [inline extensions/amp-jwplayer/0.1/amp-jwplayer.js:291]: Can we avoid adding a generic remapping of 'outstream' to 'oi7pAMI1'?

This appears to be a value that could be specified as the consent id.

File: extensions/amp-jwplayer/0.1/amp-jwplayer.js [hunk 2]
```diff
@@ -446,6 +449,14 @@ class AmpJWPlayer extends AMP.BaseElement {
           this.playlistItem_ = playlistItem;
           this.sendCommand_('getPlayedRanges');
           break;
+        case 'resize':
+          const {width, height} = detail;
+          Services.mutatorForDoc(this.getAmpDoc()).forceChangeSize(
```
Comment (kristoferbaxter) [inline extensions/amp-jwplayer/0.1/amp-jwplayer.js:454]: Forcing a size change is a rarity in AMP components. Can we instead use `attemptChangeSize`?

File: extensions/amp-jwplayer/0.1/amp-jwplayer.js (hunk 1 above)
Comment (zetagame) [reply extensions/amp-jwplayer/0.1/amp-jwplayer.js:291]: `oi7pAMI1` is our content-id for all outstream players. This substitution is simply so customers don't have to memorize an arbitrary string.

File: extensions/amp-jwplayer/0.1/amp-jwplayer.js (hunk 2 above)
Comment (zetagame) [reply extensions/amp-jwplayer/0.1/amp-jwplayer.js:454]: I've tried this in a few different ways to no avail (wouldn't resize with attemptChangeSize, no errors/warnings), but this seemed to work seamlessly. Is this something you've seen in the past? If so, how was it resolved?

Comment (choumx) [reply extensions/amp-jwplayer/0.1/amp-jwplayer.js:454]: This is UX policy for AMP. Resizing and other operations that can cause content jumping may only be done off-screen or if triggered by a user gesture.

**Pull request closed without merging**